erpnext/loan_management/doctype/loan_disbursement/loan_disbursement.json

```diff
@@ -0,0 +1,110 @@
+{
+ "autoname": "LD-.#####",
```

Comment:
Let all loan series start with **LM**, since loan management is now a module. example: `LM-DIS-.###`